src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,232 @@
+package Duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+// import Duke.Function.*;
+// import Duke.Task.*;
+import java.util.Scanner;
+// import java.util.List;
+
+public class Duke {
+    private static final String VERSION = "Duke - Version 1.0";
+    private static final String DIVIDER = "===================================================";
+    private static final String LOGO = " ____        _        \n" + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n" + "| |_| | |_| |   <  __/\n" + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String GREETING = "Hello! I'm Duke\n" + "What can I do for you?\n";
+
+    private static final Scanner SCANNER = new Scanner(System.in);
```

Comment:
Perhaps you might want to consider if it's best to store your Scanner as a constant, from a program functionality point-of-view.